config/initializers/sidekiq.rb

```diff
@@ -0,0 +1,9 @@
+redis_host = ENV.fetch("RAILS_REDIS_HOST", "localhost")
```

Comment:
The tricky part about redis is that this config will not work on production because redis will not be in the local host. I don't like REDIS_URL because we want the sidekiq server to be different from the action cable server.

I think we should use a full JOBS_REDIS_URL